jekyll/_cci2/ip-ranges.md

```diff
@@ -0,0 +1,201 @@
+
+The feature is currently available in preview to customers on any [paid pricing plan](https://circleci.com/pricing/).
+
+## IP Ranges: use cases
+{: #usecases }
+
+	IP Ranges empowers customers to limit inbound connections to their infrastructure to IP address ranges that are verifiably associated with CircleCI.  
+
+Some example customer use cases where IP-based restricted access might be desired include:
+- Accessing private artifact repositories 
+- Pulling dependencies from a CocoaPods proxy hosted behind a firewall
+- Running test cases on an internal environment 
+- Performing integration testing against private AWS resources
+- Deploying an internal app with sensitive data
+- Granting access to a production network 
+
+Previous to offering IP Ranges, the only solution CircleCI offered customers in need of static IP addresses which could be configured and controlled was [CircleCI’s Runner](https://circleci.com/docs/2.0/runner-overview/). IP Ranges now enables customers to meet their IP-based security and compliance requirements using their existing workflows and platform. 
+
+## Example configuration file that uses IP Ranges
+{: #exampleconfiguration }
+
+```
+version: 2.1
+jobs:
+  build:
+    circleci_ip_ranges: true # opts the job into the IP Ranges feature
+    docker:
+      - image: curlimages/curl
+    steps:
+      - run: echo “Hello World”
+workflows:
+  build-workflow:
+    jobs:
+      - build
+```
+
+## List of IP address ranges associated with the IP Ranges feature
+{: #listofipaddressranges }
+
+*Last updated*: 07/21/2021
+
+Jobs that have opted into the IP Ranges feature will have one of the following IP address ranges associated with them:
+
+- 107.22.40.20
+- 18.215.226.36
+- 3.228.208.40
+- 3.228.39.90
+- 3.91.130.126
+- 34.194.144.202
+- 34.194.94.201
+- 35.169.17.173
+- 35.174.253.146
+- 52.20.179.68
+- 52.21.153.129
+- 52.22.187.0
+- 52.3.128.216
+- 52.4.195.249
+- 52.5.58.121
+- 52.72.72.233
+- 52.72.73.201
+- 54.144.204.41
+- 54.161.182.76
+- 54.162.196.253
+- 54.164.161.41
+- 54.167.72.230
+- 54.205.138.102
+- 54.209.115.53
+- 54.211.118.70
+- 54.226.126.177
+- 54.81.162.133
+- 54.83.41.200
+- 54.92.235.88
+
+IP address ranges for core services (used to trigger jobs, exchange information about users between CircleCI and Github etc):
+
+- 18.214.70.5
+- 52.20.166.242
+- 35.174.249.131
+- 18.214.156.84
+- 54.236.156.101
+- 3.210.128.175
+
+**Upcoming Changes to List of IP Address Ranges:
+
+*Last Updated*: 07/21/2021
+
+None
+
+Note that jobs can use any of the address ranges above.  It’s also important to note that the address ranges are shared by all CircleCI customers who have opted into using the feature. **can we make this a note like we do in the runner overview pag?**
```

Comment:
The runner docs are written in asciidoc, which provides enhanced admonition features, you can get a similar effect with ` {: class="alert alert-warning"}` on the line below I think though...

Reply:
I can push a change to see if you like!

Reply:
that would be great, thank you!

Reply:
is there a way to see a preview of the docs? i thought i saw in some of the other guidance that if i added "-preview" it would spin up a preview page? or am i misremembering

Reply:
Yes there's a job in the build deploy workflow called deploy-preview that gives a URL

Reply:
I can't see the job so I'm wondering if it's not running because this is a fork PR 🤔

Reply:
ah interesting.  should i do something to change that? i thought that the guidance was to create a fork, should i have done it a different way? sorry this is my first time making major changes so just want to make sure i know the best practices going forward 😬

Reply:
... so yes in the circleci app when I look at your branch it doesn't show your actual branch name, but instead the PR number, so the `-preview` is not there. This must be why the preview site isn't being built. I will talk to the team about it so we can make sure the contributor's guide is correct. Once we're happy with these changes could make a new branch not on a fork to preview for formatting etc.